airgun/entities/computeresource.py

```diff
@@ -33,6 +34,12 @@ def delete(self, value):
         view.table.row(name=value)['Actions'].widget.fill('Delete')
         self.browser.handle_alert()
 
+    def list_RHEV_VMs(self, rhev_name, expected_vm_name):
```

Comment:
1) why not `list_vms`? and in any case - use lowercase.
2) make expected_vm_name optional (`expected_vm_name=None`)
3) pls add docstring
4) `else view.virtual_machines.table.row()` why just single row() and not rows()? Helpers is called list VMs, not just 1 vm.